ONSdigital/ras-frontstage
# RAS-1035: Spike log based labelled metric for frontstage send message (best solution)

# What and why?
There is a requirement to create a metric that shows a visualisation of the secure messages sent by Frontstage. This PR accomplishes this.

# How to test?
- Run unit tests
- Deploy this PR to your dev env
- Run the acceptance tests
- Load up the metric labelled Frontstage-secure-messages-sent in dev metrics explorer
- Send a variety of messages from Frontstage
- Refresh the metric to see the graphs formed

# Jira
https://jira.ons.gov.uk/browse/RAS-1035
